src/WatchDogs.Contracts/ISuspiciousDealDetector.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WatchDogs.Contracts;
+public interface ISuspiciousDealDetector
+{
+    Task<List<Trade>> LoadDealsAsync();
```

Comment:
Tbh I think only method in this interface should be `DetectSuspiciousDealsAsync`. Loading is CRUD related concern and so is sorting

Reply:
Okay.